specification/quota/resource-manager/Microsoft.Quota/preview/2019-07-19-preview/quota.json

```diff
@@ -0,0 +1,609 @@
+        "operationId": "Quota_Request_Submit",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ProviderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/QuotaRequestNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "createQuotaRequests",
+            "in": "body",
+            "description": "Quota requests payload.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CreateGenericQuotaRequestParameters"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Quota requests will be created asynchronously. A request id for each changed resource is returned in the same order as the order in which they were provided.",
+            "schema": {
+              "$ref": "#/definitions/QuotaRequestResponseStatus"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "QuotaRequests"
+        ],
+        "summary": "For the specified location, gets the Quota request status by request name provided during the quota request creation for the specific resources under the subscription.",
+        "description": "Gets the QuotaRequest details and status by the quota Request Name for the resources for the resource  provider at a specific location.",
+        "operationId": "QuotaRequests_Status_Check",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ProviderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/QuotaRequestNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok. Returning the status for all relevant quota requests .",
+            "schema": {
+              "$ref": "#/definitions/QuotaRequestDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
Why is this set to null?  The response type defines a `nextLink` field.

Reply:
/subscriptions/{subscriptionId}/providers/Microsoft.Quota/providers/{providerId}/locations/{locationId}/quotaRequests/{requestName} is for single request.
I will update the response to quotaRequestDetail and will remove QuotaRequestDetails.

Reply:
Updated the response.